themes/default/views/administrate/setup/interface_editor/screen_html.php

```diff
@@ -25,33 +25,35 @@
  *
  * ----------------------------------------------------------------------
  */
- 	$t_subject = $this->getVar('t_subject');
```

Comment:
Just meaning, by completely changing the formatting, we're making it obvious that we've rewritten this particular view, so it indicates that it's "ours" as opposed to "theirs" (upstream).  But we have probably changed other views as well without reformatting, so it's not a perfect indicator.